udata/migrations/2023-02-08-rename-internal-dates.py

```diff
@@ -15,13 +16,23 @@ def migrate(db):
     dataset_collection = db.dataset
     dataset_collection.update_many({}, {'$rename': {
         'created_at': 'created_at_internal',
-        'last_modified': 'last_modified_internal',
-        'resource.created_at': 'resource.created_at_internal',
-        'resource.modified': 'resource.last_modified_internal'}})
+        'last_modified': 'last_modified_internal'}})
 
     community_resource_collection = db.community_resource
     community_resource_collection.update_many({}, {'$rename': {
         'created_at': 'created_at_internal',
-        'modified': 'last_modified_internal'}})
+        'modified': 'last_modified_internal',
+        'harvest.modified_at': 'harvest.last_modified'}})
```

Comment:
Is that needed? Community ressources are never harvested right? So I guess not any will have the field set?

Reply:
Indeed, we don't have community resource with an harvest field :)